globals.go

```diff
@@ -508,6 +508,7 @@ const (
 	TaskExecutionOtelAttribute   = "evergreen.task.execution"
 	TaskStatusOtelAttribute      = "evergreen.task.status"
 	TaskFailureTypeOtelAttribute = "evergreen.task.failure_type"
+	TaskTagsSliceOtelAttribute   = "evergreen.task.tags"
```

Comment:
nice: I don't think we need to mention that it's a slice in this? None of the others mention the attribute's type.